addon-test-support/-private/utils.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+export function checkMatcher(message, matcher) {
+  if (typeof matcher === 'string') {
+    return message.includes(matcher);
```

Comment:
HAHA, I think my thought process was exactly the opposite of yours.  A regexp is already "magical" and can support partial matches, whereas its quite annoying/hard to match the _exact_ message with a regexp (e.g. the escaping for periods, brackets, etc).

I do not care strongly though, and I think your argument of it being a breaking change is a totally reasonable reason to not change the default.